README.md

```diff
@@ -10,5 +10,18 @@ Azure Communication Services enable developers to add communication capabilities
 
 ## Quickstarts
 
+### Calling SDK Quickstarts
 1. [Add Voice Calling to your Android App](https://docs.microsoft.com/azure/communication-services/quickstarts/voice-video-calling/getting-started-with-calling?pivots=platform-android)
-2. [Add telephone calling to your Android App](https://docs.microsoft.com/azure/communication-services/quickstarts/voice-video-calling/pstn-call?pivots=platform-android)
+2. [Add video calling to your Android App](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/get-started-with-video-calling)
+3. [Add video calling to your WebView Android App](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/get-started-android-webview)
+4. [Add telephone calling to your Android App](https://docs.microsoft.com/azure/communication-services/quickstarts/voice-video-calling/pstn-call?pivots=platform-android)
+5. [Record Calls](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/get-started-call-recording)
+6. [Handle Calling Events](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/handle-calling-events)
+7. [Raw media Access](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/get-started-raw-media-access)
+
+### Chat SDK Quickstarts
+1. [Add Chat to our App](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/chat/get-started)
+2. [Using Chat SDK with React Native](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/chat/react-native)
+
+### UI Libraries
+1. [Getting Started with the UI Libraries](https://learn.microsoft.com/en-us/azure/communication-services/quickstarts/ui-library/get-started-composites?tabs=kotlin&pivots=platform-android)
```

Comment:
The url will change once we publish the MSdoc. It will be `get-started-calling-ui-library` instead of `get-started-composites`

Reply:
@Leah-Xia-Microsoft name of the page is changed in left hand menu, but URL remain the same (for now)